README.md

```diff
@@ -65,7 +65,7 @@ dotnet restore
 dotnet watch run
 
 # 2: webpack frontend
-cd src
+cd src/Blaster.WebApi
```

Comment:
Any particular reason for this change? As far as I can see, there is no difference in behaviour when 'npm start' is called. My current assumption is that it has been done for just mere semantics, would I be wrong in that assumption?

Reply:
The content root folder becomes src and not src/Blaster.Webapi which will cause 404 errors when trying to fetch css and js files